integration_tests/fixtures.py

```diff
@@ -0,0 +1,172 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from future import standard_library
+standard_library.install_aliases()
+
+import logging
+import os
+import os.path
+import shutil
+import signal
+import socket
+import subprocess
+import tempfile
+import time
+
+import pytest
+
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+PRESTO_VERSION = os.environ.get('PRESTO_PYTHON_CLIENT_PRESTO_VERSION', '0.157')
+PRESTO_URL_FORMAT = 'https://repo1.maven.org/maven2/com/facebook/presto/presto-server/{v}/presto-server-{v}.tar.gz'.format(v=PRESTO_VERSION)  # NOQA
+TEMPDIR_PREFIX = 'tests-presto-python-client.'
+# Set the environment variable below to cache Presto tarballs.
+# Beware of security issues when using files from /tmp (or other shared
+# directories) as someone else can leave there a modified version of Presto.
+PRESTO_PYTHON_CLIENT_TEST_CACHE_DIR = os.environ.get(
+    'PRESTO_PYTHON_CLIENT_TEST_CACHE_DIR',
```

Comment:
Remove the trailing comma since there's always one arg:
```python
PRESTO_PYTHON_CLIENT_TEST_CACHE_DIR = os.environ.get(
    'PRESTO_PYTHON_CLIENT_TEST_CACHE_DIR')
```
Also, the local variable name is quite long. It could just be `CACHE_DIR`:
```python
CACHE_DIR = os.environ.get('PRESTO_PYTHON_CLIENT_TEST_CACHE_DIR')
```